src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -313,19 +307,21 @@ public synchronized ListeFilme neueFilme(ListeFilme orgListe) {
         return ret;
     }
 
-    public String getFileSizeUrl(String url, String sender) {
+    //FIXME bring to DatenFilm and reduce calculation
+    @Deprecated
+    public String getFileSizeUrl(String url) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1133)